frontend/sw360-portlet/src/main/webapp/html/components/edit.jsp

```diff
@@ -155,6 +157,20 @@ require(['jquery', 'modules/sw360Validate', 'modules/autocomplete', 'modules/con
 
         $('#formSubmit').click(
             function () {
+                var valid = true;
+                $.each($('#externalIdsTable tr.bodyRow'), function (i, row) {
+                    var key = $('#'+row.id+' td input.keyClass').val();
+                    var value = $('#'+row.id+' td input.valueClass').val();
+                    var isOneEmpty = ((key !== '') && (value === '')) || ((key === '') && (value !== ''));
+                    if (isOneEmpty) {
+                        $.alert('There is a blank field in one or more of your external ids. Please correct this before updating the component.');
+                        valid = false;
+                        return false;
```

Comment:
returned value is ignored. Instead an environmental variable is modified. That looks not optimal.

Reply:
js removed and added required/minlength to the external id input fields